app/views/users/totp_setup/index.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('upaya.titles.enter_2fa_code')
+
+.clearfix.mxn2
+  .sm-col-6.px2.mx-auto
+    .panel
+      .panel-heading
+        h2.m0 = t('upaya.forms.totp_setup.header_text')
+
+      p
+        'To enable app-based authentication please scan the QR code below
+        'and into your TOTP-compatiblre mobile app and then enter the code
+        'generated in the box below.
```

Comment:
How about `To enable app-based authentication, please use your TOTP-compatible mobile app to scan the QR code below, then enter the code displayed in your app in the box below`?